### ISSUE TITLE
Add support for multiple space separated keys

### ISSUE THREAD

**Issue by fatih** [fatih/structtag#13]
There is a new feature coming soon to Go 1.6 where one can define multiple keys to a single value.  

Before

```go
type MyStruct struct {
  Field1 string `json:"field_1,omitempty" bson:"field_1,omitempty" xml:"field_1,omitempty" other:"value"`
}
```

After

```go
type MyStruct struct {
  Field1 string `json bson xml:"field_1,omitempty" other:"value"`
}
```

We need to update the `Parse` to accommodate the new changes.  

CL for the Implementation in the reflect package: https://go-review.googlesource.com/c/go/+/248341/ 
CL for the doc: https://go-review.googlesource.com/c/go/+/274448/

**Comment by joshuabezaleel** [comment]
Hi @fatih , a big fan of you from Twitter! Stumbled upon your tweet on this particular issue. I am not that experienced of Go programmer but may I have a shot on this? 🙂

**Comment by fatih** [comment]
Hi @joshuabezaleel 

I saw that you already created a PR. First thank you for your contribution. I'm nowadays dealing with some life changing events and hence don't have the bandwidth to look at it. 

First of all, I already had a working implementation locally. However, I didn't pushed it yet as this syntax _may_ change. Checkout the discussion here: https://github.com/golang/go/issues/40281#issuecomment-746770550 Hence let's wait until this has finalized. I might close your PR or might push some changes if things finalize and we have a proper roadmap.

Second, there are couple of things we need to make sure. It's not just parsing the keys, we also need to associate multiple keys with each other. Unfortunately, the current API is not suitable for it (unless we do some ugly changes, like treating `Key string` a comma separated tag, such as `json,xml,bson`). So we either need to find a way to introduce multiple keys in an intuitive way OR we need to introduce a breaking change, which means we need to publish a new `github.com/fatih/structttag/v2` version.

Thanks again working on that, again let's wait until we got something we can work on.
